README.md

```diff
@@ -36,7 +36,12 @@ The top level of the configuration file looks like this:
   ...
   "snaps": [
     {
+      "body": { ... }
```

Comment:
I think you’re missing a comma at the end of this line. 😄

Reply:
thx :)